zuul-core/src/main/java/com/netflix/zuul/exception/OutboundErrorType.java

```diff
@@ -55,6 +55,10 @@ public enum OutboundErrorType implements ErrorType {
             ERROR_TYPE_ORIGIN_RESET_CONN_STATUS.get(),
             ZuulStatusCategory.FAILURE_ORIGIN_RESET_CONNECTION,
             ClientException.ErrorType.CONNECT_EXCEPTION),
+    CLOSE_NOTIFY_CONNECTION(
+            504,
```

Comment:
nit: This is technically the equivalent of `CONNECT_ERROR` imo, which maps to a `502` vs `504`

Reply:
Updated 👍